src/binaries/Cargo.toml

```diff
@@ -47,8 +47,9 @@ databend-meta = { path = "../meta/service" }
 databend-query = { path = "../query/service" }
 
 # Crates.io dependencies
-anyhow = "1.0.58"
+anyhow = "1.0.64"
 clap = { version = "3.2.5", features = ["derive", "env"] }
+openraft = { git = "https://github.com/datafuselabs/openraft", rev = "6057abbf" }
```

Comment:
A `rev` is unstable: a commit can be removed by the author.

The tag won't be removed(by convention). And a tag name is also more clear for developers.

Reply:
Addressed